tests/test_builder.py

```diff
@@ -0,0 +1,222 @@
+import unittest
+from typedbSchemaBuilder import Builder
```

Comment:
This import is incorrect, how do you run tests?
```suggestion
from typedbSchemaBuilder import builder
```